server/game/cards/12-SoW/AnOceanInADrop.js

```diff
@@ -0,0 +1,51 @@
+const DrawCard = require('../../drawcard.js');
+const { Locations, TargetModes } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl.js');
+
+class AnOceanInADrop extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Place hand on bottom of deck and draw cards',
+            condition: context => this.game.isDuringConflict() && context.source.parent && context.source.parent.isParticipating(),
```

Comment:
`this.game.isDuringConflict()` seems superfluous here.  You also shouldn't need to null check parent